developer_guides/debugability/logger/index.rst

```diff
@@ -136,6 +136,11 @@ Examples
     distribution kernels ``sof_debug=1`` module option for ``snd_sof`` might be
     needed if the /sys/kernel/debug/sof/trace file is not present)
 
+.. note::
+  The DMA trace sysfs interface is a streaming interface. If sof-logger is
+  started (or restarted) while DSP is active, it is possible the initial traces
```

Comment:
typo: s/traces/trace

Reply:
Would the following work?

The DMA trace sysfs interface is a streaming interface. If sof-logger is started (or restarted) while DSP is active, the initial trace messages might appear in the wrong order or incomplete.